app/models/bet.rb

```diff
@@ -27,4 +23,28 @@ def amount_owed(lost_bet, won_bet)
   def bet_total
     user_bets.pluck(:amount_bet).sum
   end
+
+  private
+
+  def resolve_debt(lost_bet, won_bet)
+    existing_debt = Debt.find_by(debtor: lost_bet.user, creditor: won_bet.user)
+    if existing_debt
+      update_debt(existing_debt, lost_bet, won_bet)
+    else
+      create_debt(lost_bet, won_bet)
+    end
+  end
+
+  def create_debt(lost_bet, won_bet)
+    Debt.create(
+      debtor: lost_bet.user,
+      creditor: won_bet.user,
+      amount:  amount_owed(lost_bet, won_bet)
+    )
+  end
+
+  def update_debt(debt, lost_bet, won_bet)
```

Comment:
This is called feature creep. Its funtionality that was not specified in any story and should not have been implemented (yet)